### PR TITLE
Raidboss: Change directions to Outputs

### PR DESCRIPTION
**Pull request by MaikoTan**
@ShadyWhite @Jaehyuk-Lee 
There're some inconsistent after I made this change. Although I think it is good.
May you all review it if it is ok?

### CODE REVIEW

**Review by ShadyWhite — APPROVED** [review]
Looks good for me.

File: ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js [hunk 1]
```diff
@@ -2930,38 +2931,19 @@ export default {
...
-          de: 'Osten',
-          fr: 'Est',
-          ja: '東へ',
-          cn: '东',
-          ko: '동쪽으로',
-        },
+        north: Outputs.north,
+        east: Outputs.east,
         south: {
+          // It is a bit different in JA/CN that players should go middle instead of south,
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js:2937]: That is extremely unfortunate that these call different things in different languages.

**Comment by Jaehyuk-Lee** [comment]
@Una-p I'm very busy until tomorrow, could you review these please? 좀 생각을 해봐야 하는 문제인데 약 이틀간 PC도 못쓰고 오늘까지 해야 하는 일도 있고 해서요.

**Comment by Una-p** [comment]
Looks good for us, too. 
And TEA south direction note is same as ko. That translated [middle].

**Comment by MaikoTan** [comment]
OK, Thank you all!

@quisquous I think it is good to merge now!

**Pull request merged**